clients/thymioupgrader/ThymioUpgrader.cpp

```diff
@@ -58,7 +58,7 @@ namespace Aseba
 	{
 		// Create the gui ...
 		setWindowTitle(tr("Thymio Firmware Upgrader"));
-		
+        setWindowIcon(QIcon(":/images/thymioupgrader.svgz"));
```

Comment:
please use tabs (representing 4 spaces) to be consistent with the existing code base